pkg/kafka/consumer/reader_test.go

```diff
@@ -14,9 +14,9 @@ import (
 var (
 	readerDialer = &kafka.Dialer{ClientID: "my-client"}
 	readerConf   = (&consumer.Settings{
```

Comment:
same here  
I know it's nitpicking but just for the consistency.. :)

Reply:
it makes sense